lesson_05/homework_05.py

```diff
@@ -1,39 +1,77 @@
 small_list = [3, 1, 4, 5, 2, 5, 3]
 big_list = [3, 5, -2, -1, -3, 0, 1, 4, 5, 2]
 # task 1. Знайдіть всі унікальні елементи в списку small_list
+print("Всі унікальні елементи в списку small_list:", set(small_list))
 
 # task 2. Знайдіть середнє арифметичне всіх елементів у списку small_list
+average_value = sum(small_list) / len(small_list) # или sum(small_list) // len(small_list) чтоб без десятичных
+print("Cереднє арифметичне всіх елементів у списку small_list:", average_value)
 
 # task 3. Перевірте, чи є в списку big_list дублікати
-
+length_list = (len(big_list))
+unique_values = set(big_list)
+duplicates = (length_list == unique_values)
+if duplicates != True :
+    print("дублікати є")
+else:
+    print("дублікатів нема")
 
 base_dict = {'contry':'Ukraine', 'continent': 'Europe', 'size': 123}
 add_dict = {"a":1, "b":2, "c":2, "d":3, 'size': 12}
 # task 4. Знайдіть ключ з максимальним значенням у словнику add_dict
+print("ключ з максимальним значенням у словнику add_dict:", max(add_dict.values()))
 
 # task 5. Створіть новий словник, в якому ключі та значення будуть
 # замінені місцями у заданому словнику
+new_dict = {}
+for key, value in add_dict.items():
+  new_dict[value] = key
+print("Новий словник, в якому ключі та значення замінені місцями у заданому словнику:",new_dict)
 
 # task 6. Об'єднайте два словника base_dict та add_dict  в новий словник sum_dict
 # Якщо ключі збігаються, то перетворіть значення в строку та об'єднайте їх
+
 sum_dict = {}
+sum_dict = (base_dict | add_dict)
+sum_dict = list(zip(base_dict.values(), add_dict.values()))
```

Comment:
Тут не докінця добили, треба було якось так
```suggestion
keys = set(base_dict.keys() | add_dict.keys())
values = set(base_dict.values() | add_dict.values())
sum_dict = dict(zip(keys , values)))
```
але навіть так ще не хватає if  на дублікати